src/UNL/Officefinder.php

```diff
@@ -115,22 +114,7 @@ public function __construct($options = [])
     protected static function _getAuth()
     {
         if (!self::$auth) {
-            $protocol = new SimpleCAS_Protocol_Version2([
-                'hostname' => 'login.unl.edu',
-                'port' => 443,
-                'uri' => 'cas',
-                'request' => new HTTP_Request2(null, null, [
-                    'adapter' => 'HTTP_Request2_Adapter_Curl'
-                ]),
-            ]);
-            $cacheDriver = new \Stash\Driver\FileSystem();
-            $cacheDriver->setOptions([
-                'path' => realpath(__DIR__ . '/../..') . '/tmp/simpleCAS_map',
-            ]);
-            $sessionMap = new SimpleCAS_SLOMap($cacheDriver);
-            $protocol->setSessionMap($sessionMap);
-
-            self::$auth = SimpleCAS::client($protocol);
+            self::$auth = new UNL_Officefinder_Auth();
```

Comment:
Hmm. Perhaps a better pattern for a singleton would be something like `UNL_Officefinder_Auth::getInstance()` rather than putting construction logic in another class, but both of these are out of scope of the current PR.